repository/snowflake.go

```diff
@@ -180,6 +180,9 @@ func (s *Snowflake) GetTrackingData(
 	}
 
 	rows, err := tx.QueryContext(ctx, buildGetTrackingData(trackingTable, fields, offset, limit))
+	if rows.Err() != nil {
+		return nil, fmt.Errorf("run query: %w", rows.Err())
+	}
```

Comment:
I guess a linter complained? Anyway, at this point, `rows` can be nil (if `err` is not nil), so rows.Err() would panic in that case. This check can be moved below the err != nill check.